Tests/MongoSwiftSyncTests/UnifiedTestRunner/SpecialTestOperations.swift

```diff
@@ -143,4 +267,23 @@ struct UnifiedTargetedFailPoint: UnifiedOperationProtocol {
     static var knownArguments: Set<String> {
         ["failPoint", "session"]
     }
+
+    func execute(on _: UnifiedOperation.Object, context: Context) throws -> UnifiedOperationResult {
```

Comment:
could we use the `FailPoint` type here instead of manual runCommand? It has a `Decodable` implementation so we can swap out the `BSONDocument` directly. As another improvement, we can add a new `FailPointGuard` type that is returned from `FailPoint.enable` which has a `deinit` that disables the enabled failpoint, and we can store those instead of the command name / address tuple. We do something similar in Rust.

Reply:
✅ ✅

Reply:
so this actually led a bunch of other tests that use the failpoint type to start failing, because I was not using the return value of `enable` so the failpoints were just being immediately deactivated. 
I don't think it's worth the time to go through and update those runners to pass around the `FailPointGuard`s and keep them alive, so I'm just going to switch to manually constructing the `FailPointGuard`s in the new operations.